app/api/datastore/entities/v1/maat/application.rb

```diff
@@ -3,11 +3,17 @@ module Entities
     module V1
       module MAAT
         class Application < BaseApplicationEntity
+          expose :declaration_signed_at
```

Comment:
There is a shortcut if you like, that avoids having to create the additional method.

```ruby
expose :submitted_at, as: :declaration_signed_at
```

It will expose any other attribute with a different name, an alias.

Give it a try just in case, but I think it should work.

Reply:
That is so much better!